packages/manager/src/features/CloudPulse/Alerts/AlertsDetail/AlertDetailNotification.tsx

```diff
@@ -0,0 +1,78 @@
+import { CircleProgress, Typography } from '@linode/ui';
+import { Divider, Grid } from '@mui/material';
+import React from 'react';
+
+import { ErrorState } from 'src/components/ErrorState/ErrorState';
+import { useAlertNotificationChannelsQuery } from 'src/queries/cloudpulse/alerts';
+
+import { convertStringToCamelCasesWithSpaces } from '../../Utils/utils';
+import { getChipLabels } from '../Utils/utils';
+import { AlertDetailRow } from './AlertDetailRow';
+import { DisplayAlertDetailChips } from './DisplayAlertDetailChips';
+
+interface NotificationProps {
+  /*
+   * The list of channel ids associated with the alert for which we need to display the notification channels
+   */
+  channelIds: number[];
+}
+export const AlertDetailNotification = (props: NotificationProps) => {
+  const { channelIds } = props;
+
+  const {
+    data: channels,
+    isError,
+    isFetching,
+  } = useAlertNotificationChannelsQuery({}, { id: channelIds.join(',') });
+
+  if (isFetching) {
+    return <CircleProgress />;
+  }
+
+  return (
+    <>
+      <Typography marginBottom={2} variant="h2">
```

Comment:
```suggestion
    <Stack gap={2}>
      <Typography variant="h2">
```

gap will be much better if we want to control the spacing between two elements instead of adjusting with margin